packages/js/src/utils/inline.ts

```diff
@@ -88,6 +88,8 @@ export function postProcessInlinedDependencies(
     }
   }
 
+  // TODO: hard-coded "libs"
```

Comment:
Same as the `/src` above

Reply:
let's use `libsDir()`?

Reply:
@nartc i could not find a function called `libsDir`, I found `getWorkspaceLayout` but there's no access to a Tree there. Could you point me to the right function? I'm gonna update the PR asap then.

Thank you!

Reply:
We should be handling this in a more generic way. FYI, the inline feature will be replaced in a future release but for now we should check if a package is already moved before attempting, rather than moving it here.